src/main/java/com/stripe/model/MetadataStore.java

```diff
@@ -14,14 +14,10 @@
  */
 public interface MetadataStore<T> {
 	Map<String, String> getMetadata();
-	void setMetadata(Map<String, String> metadata);
 
 	MetadataStore<T> update(Map<String, Object> params) throws AuthenticationException, InvalidRequestException,
 		APIConnectionException, CardException, APIException;
 
-	@Deprecated
-	MetadataStore<T> update(Map<String, Object> params, String apiKey) throws AuthenticationException, InvalidRequestException,
```

Comment:
Do we want to keep this consistent across classes? I didn't remove any uses of this, but I didn't add it to new classes, which is why I removed it from the interface.